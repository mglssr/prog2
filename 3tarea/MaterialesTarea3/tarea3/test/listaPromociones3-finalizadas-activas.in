crearListaPromociones
crearPromocion 01/01/2002 01/05/2002 10 1
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaPromociones
crearPromocion 01/02/2002 01/06/2002 10 2
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaPromociones
crearPromocion 01/03/2002 01/07/2002 10 3
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaPromociones
obtenerPromocionesFinalizadas 02/05/2002
imprimirListaPromociones
obtenerPromocionesFinalizadas 02/06/2002
imprimirListaPromociones
obtenerPromocionesFinalizadas 02/07/2002
imprimirListaPromociones
liberarListaPromociones
crearListaPromociones
crearPromocion 01/01/2002 01/07/2002 10 1
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaPromociones
crearPromocion 01/02/2002 01/07/2002 10 2
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaPromociones
crearPromocion 02/03/2002 01/07/2002 10 3
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaPromociones
imprimirListaPromociones
obtenerPromocionesFinalizadas 02/08/2002
imprimirListaPromociones
liberarListaPromociones
crearListaPromociones
crearPromocion 01/01/2002 01/07/2002 10 1
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaPromociones
crearPromocion 01/02/2002 01/07/2002 10 2
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaPromociones
crearPromocion 02/03/2002 01/07/2002 10 3
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaPromociones
obtenerPromocionesActivas 15/01/2002
imprimirListaPromociones
obtenerPromocionesActivas 15/02/2002
imprimirListaPromociones
obtenerPromocionesActivas 15/03/2002
imprimirListaPromociones
liberarListaPromociones
crearListaPromociones
crearPromocion 01/01/2002 01/07/2002 10 1
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaPromociones
crearPromocion 01/02/2002 01/07/2002 10 2
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaPromociones
crearPromocion 02/03/2002 01/07/2002 10 3
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaPromociones
obtenerPromocionesActivas 15/03/2002
imprimirListaPromociones
liberarListaPromociones
Fin
