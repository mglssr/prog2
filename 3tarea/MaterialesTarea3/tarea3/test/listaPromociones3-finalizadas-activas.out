Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> Lista de promociones creada con exito.
2> Promocion creada con exito.
3> Productos agregados a la promocion de forma exitosa.
4> Se ha agregado la promocion a la lista de forma exitosa.
5> Promocion creada con exito.
6> Productos agregados a la promocion de forma exitosa.
7> Se ha agregado la promocion a la lista de forma exitosa.
8> Promocion creada con exito.
9> Productos agregados a la promocion de forma exitosa.
10> Se ha agregado la promocion a la lista de forma exitosa.
11> Promociones finalizadas: 
Promocion #1 del 1/1/2002 al 1/5/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
12> Promocion #2 del 1/2/2002 al 1/6/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
Promocion #3 del 1/3/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
13> Promociones finalizadas: 
Promocion #2 del 1/2/2002 al 1/6/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
14> Promocion #3 del 1/3/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
15> Promociones finalizadas: 
Promocion #3 del 1/3/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
16> 17> Lista promociones liberada con exito.
18> Lista de promociones creada con exito.
19> Promocion creada con exito.
20> Productos agregados a la promocion de forma exitosa.
21> Se ha agregado la promocion a la lista de forma exitosa.
22> Promocion creada con exito.
23> Productos agregados a la promocion de forma exitosa.
24> Se ha agregado la promocion a la lista de forma exitosa.
25> Promocion creada con exito.
26> Productos agregados a la promocion de forma exitosa.
27> Se ha agregado la promocion a la lista de forma exitosa.
28> Promocion #1 del 1/1/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
Promocion #2 del 1/2/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
Promocion #3 del 2/3/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
29> Promociones finalizadas: 
Promocion #1 del 1/1/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
Promocion #2 del 1/2/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
Promocion #3 del 2/3/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
30> 31> Lista promociones liberada con exito.
32> Lista de promociones creada con exito.
33> Promocion creada con exito.
34> Productos agregados a la promocion de forma exitosa.
35> Se ha agregado la promocion a la lista de forma exitosa.
36> Promocion creada con exito.
37> Productos agregados a la promocion de forma exitosa.
38> Se ha agregado la promocion a la lista de forma exitosa.
39> Promocion creada con exito.
40> Productos agregados a la promocion de forma exitosa.
41> Se ha agregado la promocion a la lista de forma exitosa.
42> Promociones activas: 
Promocion #1 del 1/1/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
43> Promocion #2 del 1/2/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
Promocion #3 del 2/3/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
44> Promociones activas: 
Promocion #2 del 1/2/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
45> Promocion #3 del 2/3/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
46> Promociones activas: 
Promocion #3 del 2/3/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
47> 48> Lista promociones liberada con exito.
49> Lista de promociones creada con exito.
50> Promocion creada con exito.
51> Productos agregados a la promocion de forma exitosa.
52> Se ha agregado la promocion a la lista de forma exitosa.
53> Promocion creada con exito.
54> Productos agregados a la promocion de forma exitosa.
55> Se ha agregado la promocion a la lista de forma exitosa.
56> Promocion creada con exito.
57> Productos agregados a la promocion de forma exitosa.
58> Se ha agregado la promocion a la lista de forma exitosa.
59> Promociones activas: 
Promocion #1 del 1/1/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
Promocion #2 del 1/2/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
Promocion #3 del 2/3/2002 al 1/7/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
60> 61> Lista promociones liberada con exito.
62> Fin.
